src/sentry/api/utils.py

```diff
@@ -40,3 +42,18 @@ def get_date_range_from_params(params):
             raise InvalidParams('start must be before end')
 
     return (start, end)
+
+
+def get_release(organization, version, projects=None):
```

Comment:
Should this be a class/static method on the Release model? :bike::house:

Reply:
I was thinking about this, but I don't think so. It's really just an API concern, the rest of the codebase doesn't need to know about the double encoding

Reply:
That said, I don't know that utils is a great place either...

Reply:
Personally I lean towards 'utils' should never be used, but we're way past that point.